xonsh/environ.py

```diff
@@ -366,9 +366,7 @@ def __str__(self):
         return str(self._d)
 
     def __repr__(self):
-        return "{0}.{1}(...)".format(
-            self.__class__.__module__, self.__class__.__name__, self._d
-        )
+        return "{0}.{1}(...)".format(self.__class__.__module__, self.__class__.__name__)
```

Comment:
Why did you change the `__repr__` here? Does this belong to this patchset?

Reply:
I was getting flake8 F523 "'...'.format(...) has unused arguments" errors.  Same for change below.

Reply:
Yes, makes sense.